subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -4425,26 +4425,20 @@ static void event_stop(u32_t ticks_at_expire, u32_t remainder,
 	LL_ASSERT(!retval);
 }
 
-static u32_t event_common_prepare(u32_t ticks_at_expire,
-				  u32_t remainder,
-				  u32_t *ticks_xtal_to_start,
-				  u32_t *ticks_active_to_start,
-				  u32_t ticks_preempt_to_start,
-				  u8_t ticker_id,
-				  ticker_timeout_func ticker_timeout_fp,
-				  void *context)
+static void event_common_prepare(u32_t ticks_at_expire,
+				 u32_t remainder,
```

Comment:
Its diff that displays so, when previous line has a + prefixed with no whitespace to compensate for.

Check the raw file: https://github.com/cvinayak/zephyr/blob/de77222b5daaadc3c1760badd1312a3f60e7f98c/subsys/bluetooth/controller/ll_sw/ctrl.c#L4429